doc/util/versionButton.php

```diff
@@ -106,7 +106,7 @@ function dropSetup() {
   // Choose button color
   if (chplTitle.includes("pre-release")) {
     button.classList.add("preRelease");
-  } else if (chplTitle != chplVersions[1]) {
```

Comment:
These changes look fine to me, but we should make sure to check the generated docs before publishing.

Reply:
@benharsh: Tony built the docs in a sandbox and it looked good.  I suggested the embedding of the "current" release in the conditional just to make things a little more bulletproof (and not that much less maintainable since we'll be searching for those version numbers when bumping these things anyway).

(I also wonder whether this change is moot from the perspective of the release since that menu doesn't really work as intended without being on something that can interpret PHP and has access to the web, but it still seemed better to do it this way than make it be orange or introduce 1.18 prematurely).

Reply:
Hi, I'll take that as "approved".